popsycle/synthetic.py

```diff
@@ -1361,34 +1508,38 @@ def _calc_event_cands_radius(bigpatch, timei, radius_cut):
         Coordinates of all the stars.
     """
     # Propagate r, b, l positions forward in time.
-    r_t = bigpatch[col_idx['rad']] + timei * bigpatch[col_idx['vr']] * kms_to_kpcday # kpc
-    b_t = bigpatch[col_idx['glat']] + timei * bigpatch[col_idx['mu_b']] * masyr_to_degday # deg
-    l_t = bigpatch[col_idx['glon']] + timei * (bigpatch[col_idx['mu_lcosb']] / np.cos(np.radians(bigpatch[col_idx['glat']]))) * masyr_to_degday # deg
-    
+    r_t = bigpatch[col_idx['rad']] + timei * bigpatch[
```

Comment:
minor: I'm not a fan of these line breaks.

Reply:
fixed